smarts/core/smarts.py

```diff
@@ -340,10 +340,32 @@ def _setup_bullet_client(self, client: bc.BulletClient):
         client.setGravity(0, 0, -9.8)
 
         plane_path = self._scenario.plane_filepath
+
+        bounding_box_lenght = (
+            self._scenario.bounding_box[2] - self._scenario.bounding_box[0]
+        )
+        bounding_box_width = (
+            self._scenario.bounding_box[3] - self._scenario.bounding_box[1]
+        )
+        bounding_box_center = [
+            (self._scenario.bounding_box[0] + self._scenario.bounding_box[2]) / 2,
+            (self._scenario.bounding_box[1] + self._scenario.bounding_box[3]) / 2,
+            0,
+        ]
```

Comment:
Can you move this up to the Scenario.bounding_box() method?

The return can be `bbox, width, height`, or `bbox_origin, width, height`.